### DISCUSSION TITLE
Multi-Day events

### DISCUSSION THREAD

**Discussion by muhammadsaeedparacha** [wdelfuego/nova-calendar#7] [Ideas & Feature Requests]
Upvote.

One of the most elegantly designed repositories. Thankyou so much

**Comment by wdelfuego** [comment]
Thanks so much! :)

This will be the first new feature, I’m halfway with it. Coming in version 1.1, hopefully this month :)

**Comment by wdelfuego** [comment]
I have just released version 1.1 with support for multi-day events :).
